syntax="proto3";

package OpWire;
option java_package="OpWire";

message Operation {
	oneof op_type {
		op_put put = 1;
		op_get get = 2;
		op_quit quit = 3;
	}

	message op_put {
		uint64 key = 1;
		bytes value = 2;
	}

	message op_get {
		uint64 key = 1;
	}

	message op_quit {
		string msg = 1;
	}
}

message Response {
	double response_time = 1;
	string err = 2;
	string msg = 3;
}

	

	


